test/hack/cleanup/main.go

```diff
@@ -105,6 +132,22 @@ func main() {
 	if err := fireMetric(ctx, cloudWatchClient, "LaunchTemplatesDeleted", float64(deleted)); err != nil {
```

Comment:
Side task: can we configure these metrics to fire to timestream, since we are using this for our metrics storage now?

Reply:
For the sake of scope, we can do that when we handle resource leak monitoring